Core/Utilities/ElasticSearchDB.py

```diff
@@ -334,20 +334,20 @@ def getUniqueValue( self, indexName, key, orderBy = False ):
       query.aggs.bucket( key,
                          'terms',
                          field = key,
-                         size = 1,
+                         size = 10000,
```

Comment:
It makes sense for readability. If you are worried about people changing it at run time, I would recommend writing a comment on top of the variable (written in capital btw ;-) ). And if you are so super worried, we can make a wrapper class that forbid changing it after initialization :)  But this is very much an overkill IMHO